src/Raven.Studio/wwwroot/App/views/database/tasks/editReplicationHubTask.html

```diff
@@ -0,0 +1,402 @@
+                                            <div class="flex-horizontal">
+                                                <span class="notBefore">
+                                                    <span class="margin-right margin-right-sm"><i class="icon-clock"></i><span>Valid From</span></span>
+                                                    <strong data-bind="text: certificate().validFromText()"></strong>
+                                                </span>
+                                                <span class="expiration margin-left">
+                                                    <span class="margin-right margin-right-sm"><i data-bind="attr: { class: certificate().expirationIcon }"></i><span>Expiration</span></span>
+                                                    <strong data-bind="attr: { class: certificate().expirationClass }, text: certificate().expirationText"></strong>
+                                                 </span>
+                                            </div>
+                                        </div>
+                                    </div>
+                                </div>
+                                <div class="input-group file-input">
+                                    <input type="file" id="certificateFilePicker" data-bind="event: { change: _.partial($root.importCertificate, $element) }" tabindex="-1" />
+                                </div>
+                                <div class="flex-horizontal certificates-actions margin-bottom" title="Import your own certificate" data-bind="if: isNewAccessItem() && !certificate() ">
+                                    <label for="certificateFilePicker" class="btn btn-default">
+                                        <i class="icon-import"></i><span>Import existing certificate</span>
+                                    </label>
+                                    <button type="button" class="btn btn-default" title="RavenDb will generate a certificate for you"
+                                            data-bind="click: $root.generateCertificate, disable: $root.spinners.generateCertificate() || $root.spinners.importCertificate(),
+                                                       css: { 'btn-spinner' : $root.spinners.generateCertificate }">
+                                        <i class="icon-magic-wand"></i><span>Generate new certificate</span>
+                                    </button>
+                                </div>
+                                <span class="help-block" data-bind="validationMessage: certificate"></span>
+                            </div>
+                        </div>
+                        <div data-bind="visible: $parent.withFiltering" class="margin-bottom margin-bottom-lg">
+                            <hr>
+                            <div class="margin-bottom margin-bottom-lg">
+                                <h3>Filter Replicated Documents by Id Prefixes</h3>
```

Comment:
Id vs ID? 

on edit document we have: 
![image](https://user-images.githubusercontent.com/1250431/91634134-478afa00-e9ee-11ea-94a4-8e6028782adf.png)

Reply:
ok - changed to ID